modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java

```diff
@@ -968,49 +968,9 @@ public JournalArticle copyArticle(
 			String targetArticleId, boolean autoArticleId, double version)
 		throws PortalException {
 
-		// Article
-
-		sourceArticleId = StringUtil.toUpperCase(
-			StringUtil.trim(sourceArticleId));
-		targetArticleId = StringUtil.toUpperCase(
-			StringUtil.trim(targetArticleId));
-
-		JournalArticle sourceArticle = journalArticlePersistence.findByG_A_V(
-			groupId, sourceArticleId, version);
-
 		if (autoArticleId) {
 			targetArticleId = String.valueOf(counterLocalService.increment());
 		}
-		else {
-			validate(targetArticleId);
-
-			if (journalArticlePersistence.countByG_A(groupId, targetArticleId) >
-					0) {
-
-				throw new DuplicateArticleIdException(
-					StringBundler.concat(
-						"{groupId=", groupId, ", articleId=", targetArticleId,
-						"}"));
-			}
-		}
-
-		User user = _userLocalService.getUser(userId);
-
-		long id = counterLocalService.increment();
-
-		long resourcePrimKey =
-			_journalArticleResourceLocalService.getArticleResourcePrimKey(
-				groupId, targetArticleId);
-
-		JournalArticle targetArticle = journalArticlePersistence.create(id);
-
-		targetArticle.setResourcePrimKey(resourcePrimKey);
-		targetArticle.setGroupId(groupId);
-		targetArticle.setCompanyId(user.getCompanyId());
-		targetArticle.setUserId(user.getUserId());
-		targetArticle.setUserName(user.getFullName());
-
-		Date modifiedDate = new Date();
 
 		ServiceContext serviceContext =
```

Comment:
You can delay the creation of the service context to inside the _copyArticle method if you know whens comes from the revert or the copy methods